luigi/server.py

```diff
@@ -214,7 +214,11 @@ def get(self, path):
 class RootPathHandler(tornado.web.RequestHandler):
 
     def get(self):
-        self.redirect("/static/visualiser/index.html")
+        visualization_graph = configuration.get_config().get('core', 'visualization_graph', '')
```

Comment:
I didn't understood what you mean about 
```
On the other hand this is in the server code. Maybe it's good to not intermix with the client logic.
```
The server.py was changed so Tornado knows what is the default url to serve index.html or index.d3.html, and this is read from the configuration so there is no client code here I think, actually regarless the configuration of the visualization_graph you can visit in your browser both

localhost:8082/index.html or localhost:8082/index.d3.html.

I will change the code().visualization_graph as you suggested, dont know what else to do here maybe you can suggest some ideas?

Let me know (or post screenshots) of your super mega diagram and let me know if it worked fine :).